src/js/types/generic.type.ts

```diff
@@ -14,7 +14,31 @@ export type Menu = GenericObject;
 
 export type Tooltip = GenericObject;
 
-export type Localization = GenericObject;
+export type StringObject = {
+    [key: string]: string;
+};
```

Comment:
I added this since there seemed to be a few use cases for it and it's more specific than `GenericObject`. I'm fine using `GenericObject` though instead and removing. Any thoughts?